FSJ_django20_project/FSJ/templates/FSJ/award.html

```diff
@@ -0,0 +1,35 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %} {# NOTE YOU MUST LOAD i18n to every HTML page even though it's also loaded to base.html #}
+
+{% block heading %}
+    {# Translators: Add award page heading #}
+    {% trans "Add Award" %}
+{% endblock %}
+
+{% block content %}
+    <div>
+        {# Translators: Greeting to the user #}        
+        {% trans "Hi" %}, {{user.first_name}} {{user.last_name}}
+    </div>
+    <br/>
+    <form action='{{ context.url }}' method='post'>
```

Comment:
A bit unsure why we're using the url from context instead of typing it directly onto this page since this template is only used in this one case. Is there some benefit to doing so? Should we transition the other templates to the same for consistency?

Reply:
This template is used for displaying the award when the coordinator tries to edit the award. The URL varies depending on which award is selected. The unique and randomly generated for each award is used in the url.

Reply:
Right, ok!